src/Contexts/ProductsContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext } from "react";
+import { useContext, useEffect, useState } from "react";
+import axios from "axios";
+const ProductsContext = createContext(null);
```

Comment:
you can use ProductContext, well the above one is having no problem. Just for better calling

Reply:
Okay, I will change.
Thankyou